tasks/in_place_upgrade.yml

```diff
@@ -0,0 +1,41 @@
+# get version numbers from all jar files found
```

Comment:
Isn't it more like "get the latest version of all the existing jar files"?

Reply:
Changed